login.html

```diff
@@ -3,7 +3,56 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Login</title>
+    <title>판다마켓 로그인</title>
+    <link rel="icon" href="images/logo/favicon.ico" />
+
+    <link
+      rel="stylesheet"
+      as="style"
+      crossorigin
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css" />
+
+    <link rel="stylesheet" href="css/global.css" />
+    <link rel="stylesheet" href="css/auth.css" />
   </head>
-  <body></body>
+
+  <body>
+    <div class="auth-container">
+      <div class="logo-container">
+        <a href="/"><img src="images/logo/logo.png" alt="판다마켓 홈" width="396" /></a>
+      </div>
+
+      <form method="post" class="auth-form">
+        <div class="input-group">
+          <label for="email">이메일</label>
+          <input id="email" name="email" type="email" placeholder="이메일을 입력해 주세요" required />
+        </div>
+        <div class="input-group">
+          <label for="password">비밀번호</label>
+          <div class="password-wrapper">
+            <input id="password" name="password" type="password" placeholder="비밀번호를 입력해 주세요" required />
+            <img src="images/login_icon/eye-invisible.svg" alt="비밀번호 감추기" class="toggle-password" />
```

Comment:
❗️ 수정요청  
해당 요소는 클릭 시 비밀번호가 보이고 안 보이게 만드는 요소가 될 것이므로 button요소로 작성하시는 것을 추천드려요.

```suggestion
            <button type="button"><img src="images/login_icon/eye-invisible.svg" alt="비밀번호 감추기" class="toggle-password" /></button>
```